HISTORY.rst

```diff
@@ -2,6 +2,13 @@
 
 Release History
 ===============
+0.10.8
++++++++++++++++
+
+**IoT Central updates**
+
+* update parsing template logic to support  DTDLV2 models.
```

Comment:
0.10.8 already has changes staged for release, combine the central notes with the existing.